client/redux/sagas/data.js

```diff
@@ -84,25 +80,25 @@ function* getFilters() {
   };
 }
 
-function* getMapFilters() {
-  const {
-    startDate,
-    endDate,
-    councils,
-    requestTypes,
-  } = yield select(getState, 'mapFilters');
-
-  const convertCouncilNameToID = ncList => (
-    ncList.map(name => COUNCILS.find(nc => nc.name === name)?.id)
-  );
-
-  return {
-    startDate,
-    endDate,
-    ncList: convertCouncilNameToID(councils),
-    requestTypes: Object.keys(requestTypes).filter(req => req !== 'All' && requestTypes[req]),
-  };
-}
+// function* getMapFilters() {
```

Comment:
I'm guessing this function isn't being used? In that case, we can just delete it. Commented-out code is more confusing than useful in most cases.

Reply:
agreed, commented code has been removed